usersApp/src/components/PostCards.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Card, CardColumns, Button } from 'react-bootstrap';
+import Users from './Users';
+import './PostCards.scss';
+
+const PostCards = ({setCards, posts, users}) => {
+  return (
+    <div className="cardContainer">
+      <CardColumns className="cardColums">
+        {posts.map((post) =>
+          <Card id="card" key={post.id} style={{ width: '16rem' }}>
+            <Card.Body>
+              <Card.Title>
+                {post.title}
+              </Card.Title>
+              <Card.Text>
+                {post.body}
+              </Card.Text>
+            </Card.Body>
+            <Card.Footer>
+                <p className="userName">
+                  Autor: {<Users user={users.filter((user) => user.id === post.userId)} />}
```

Comment:
It's better to find a user:
```suggestion
                  Autor: {<Users user={users.find((user) => user.id === post.userId)} />}
```